.github/workflows/create-release-pr.yaml

```diff
@@ -0,0 +1,66 @@
+name: Create Release PR
+
+on:
+  workflow_dispatch:
+
+jobs:
+  create_pr:
+    name: "Create Pump Version PR"
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Update Draft
+        uses: release-drafter/release-drafter@v5.15.0
+        id: draft
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setup YQ
+        uses: chrisdickinson/setup-yq@latest
+        with:
+          yq-version: v4.9.1
+
+      - name: Pars Changhelog
```

Comment:
Spelling?
```suggestion
      - name: Parse Changlog
```

Reply:
the spelling gods have not blessed me ... ever :man_facepalming: